packages/lesswrong/lib/collections/localgroups/newSchema.ts

```diff
@@ -89,6 +89,7 @@ const schema = {
   name: {
     database: {
       type: "TEXT",
+      nullable: false,
```

Comment:
Pretty sure names are supposed to in fact be required for localgroups, but again, lmk if that's not the case.

Reply:
(This was already `NOT NULL` in the db, so it's not included in the migration itself.)